playground/mocks/data/idp/idx/authenticator-verification-select-authenticator.json

```diff
@@ -464,6 +464,7 @@
         "key": "phone_number",
         "id":"pae5ykx4eIvfslkO90g4",
         "displayName":"Phone Number",
+        "nickname":"ph-nn",
```

Comment:
Preferably, we should leave the existing mock files as is, and create a new one with this nickname value. This is to showcase support for when the nickname feature is not enabled but the view still works.

Reply:
Got it. Added new mock and test.